hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/MultiMapCodecTemplate.java

```diff
@@ -266,4 +266,14 @@
      */
     @Request(id = 21, retryable = false, response = ResponseMessageConst.BOOLEAN, partitionIdentifier = "key")
     Object removeEntry(String name, Data key, Data value, long threadId);
+
+    /**
+     * Removes all the entries with the given key. Returns void
+     *
+     * @param name     Name of the MultiMap
+     * @param key      The key of the entry to remove
+     * @param threadId The id of the user thread performing the operation. It is used to guarantee that only the lock holder thread (if a lock exists on the entry) can perform the requested operation
+     */
+    @Request(id = 22, retryable = false, response = ResponseMessageConst.VOID, partitionIdentifier = "key")
```

Comment:
You may add the `@Since("1.5")` annotation here. Or is it 1.6? @ihsandemir @sancar

Reply:
@mmedenjak done.  I think 1.5 is right, I looked at other commits and it seemed others added 1.5 even when the .pom had the version already at 1.5.  LMK if I should change it.

Reply:
since 3.9 is released, can you use 1.6?

Reply:
@vmarquez  Our next release will be 1.6 so tag should be 1.6.

Reply:
@sancar ah, makes sense, I have changed to 1.6.